cli/tre/api_client.py

```diff
@@ -1,10 +1,12 @@
 import sys
+from typing import Union
 import click
 import json
 import msal
 import os
 
 from httpx import Client, Response
+from httpx._types import QueryParamTypes
```

Comment:
Looks like you're importing a "private" module. Is QueryParamTypes a stable API to depend upon?
Do you need to type it like this, or can it just be typed as a Dict?

Reply:
I used that type as that is what the `params` property on the (public) `request` method is typed as.
 
 I've updated to a `dict` now